stagecoach/jobs/publishers/checkpoint/app.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+import json
+import os
+
+import boto3
+from empiric.core.logger import get_stream_logger
+from empiric.core.utils import str_to_felt
+from empiric.publisher.assets import EMPIRIC_ALL_ASSETS
+from empiric.publisher.client import EmpiricPublisherClient
+
+logger = get_stream_logger()
+
+
+def handler(event, context):
+    invocation = asyncio.run(_handler(EMPIRIC_ALL_ASSETS))
+    return {
+        "result": invocation,
+    }
+
+
+def _get_pvt_key():
+    secret_name = "publisherPrivateKey"
```

Comment:
Should we make this key more specific e.g. version it or specify which publisher?

Reply:
we could probably give it a general name, since it can be used for multiple accounts.  I think it's ok for now, until we have another secret key in the secret manager.